packages/bitcoin/src/p2wsh-p2sh-address-gen.ts

```diff
@@ -0,0 +1,77 @@
+import { NetworkModes } from '@leather-wallet/constants';
+import { ripemd160 } from '@noble/hashes/ripemd160';
+import { sha256 } from '@noble/hashes/sha256';
+import { base58check } from '@scure/base';
+import { HDKey } from '@scure/bip32';
+import { mnemonicToSeed } from '@scure/bip39';
+import * as bitcoin from 'bitcoinjs-lib';
+
+export async function deriveBtcBip49SeedFromMnemonic(mnemonic: string) {
+  return mnemonicToSeed(mnemonic);
+}
+
+export function deriveRootBtcKeychain(seed: Uint8Array) {
+  return HDKey.fromMasterSeed(seed);
+}
+
```

Comment:
I think we should delete this code / unused exports. Now we are moving unused things to a new package to not be used more efficiently

Reply:
This seems to be used in tests tho: https://github.com/leather-wallet/mono/pull/20/files/b6c2a00e5ed088777c633b3e9406ab00c442052a#diff-0a9a7f7a5ac8ac518739538ca3867bb29f32881b8cd82345d3f0b53c5426ace1R118

Reply:
I'm talking about `deriveBtcPayment` below with the `ts-unused-exports:disable-next-line`